src/components/Home/Home.tsx

```diff
@@ -27,18 +41,67 @@ export function Home() {
       <br />
     </>
   ) : null;
+
+  const handleSubmit = () => {
+    const postalCodeRegex = /[a-zA-Z][0-9][a-zA-Z](-| |)[0-9][a-zA-Z][0-9]/;
+    if (postalCodeRegex.test(postalCode)) {
+      setRenderCards(true);
+      setShowPostalPrompt(false);
+    } else {
+      setShowInvalidPostal(true);
+    }
+  };
+  const afterPostalEntryMarkup = renderCards ? (
+    <>
+      <TextStyle variation="subdued">
+        Here&apos;s what&apos;s happening with vaccinations in Ottawa
+      </TextStyle>
+      <br />
+      <br />
+      {dismissableBannerMarkup}
+      <PharmacyList />
+    </>
+  ) : null;
```

Comment:
Don't bother doing anything about this, but I never use `null` in my code, ever. JS is weird in that it has two competing bottom types. I always use `undefined`, as that is the bottom type the language itself uses. https://www.youtube.com/watch?v=PSGEjv3Tqo0&t=561s